cmd/gnokey/add.go

```diff
@@ -0,0 +1,308 @@
+package main
```

Comment:
We should keep most of gnokey features as an importable library. Please move back most of the logic in pkgs/crypto/client.

Reply:
We’ve two goals as a library:
* allow people to craft custom clients/apps using those libraries by passing structures.
* allow people to compose chain CLIs (this package is part of tendermint2, not GnoVM).